indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -152,6 +157,8 @@ public class TaskQueue
   private final AtomicInteger statusUpdatesInQueue = new AtomicInteger();
   private final AtomicInteger handledStatusUpdates = new AtomicInteger();
 
+  private final ObjectMapper jsonMapper;
```

Comment:
Also, the `ObjectMapper` should be injected into `TaskMaster` and then passed on to `TaskQueue` where it would be copied with the required mixins (the same way as is done in `SQLMetadataStorageActionHandler`). We should not create a `new JsonMapper()` here.

Reply:
Thanks for catching this!